taxonomy/emsi_client.py

```diff
@@ -177,6 +177,32 @@ def __init__(self):
         """
         super(EMSIJobsApiClient, self).__init__(scope='postings:us')
 
+    @JwtEMSIApiClient.refresh_token
+    def get_details(self, ranking_facet, query_filter):
+        """
+        Query the EMSI API for the loopup of the pre-defined filter_query.
+
+        Arguments:
+            ranking_facet (RankingFacet): Data will be fetch for this facet.
+            query_filter (dict): Filters to be sent in the POST data.
+
+        Returns:
+            dict: A dictionary containing details of all the details.
```

Comment:
Can we change "details" to something more specific? Talking about the second "detail"